CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 - Removed: async mode/option
 ([#107](https://github.com/postcss/postcss-import/pull/107))
+- Removed: bower is not currently supported, use custom resolver
+([#116](https://github.com/postcss/postcss-import/pull/116))
+- Changed: custom resolve has more responsibility for paths resolving
```

Comment:
This message is not clear. We should explicitly tell what has changed in the signature of the function (or send the user to the right place in the doc).